clusterloader2/pkg/config/cluster.go

```diff
@@ -103,6 +103,7 @@ type PrometheusConfig struct {
 	PushgatewayManifests         string
 	StorageClassProvisioner      string
 	StorageClassVolumeType       string
+	PVCStorageClass              string
```

Comment:
I think this somehow got pulled from other commit that was merged into master

Reply:
These (storage class-related) changes are already in the master and will not appear on the merge commit I believe.

Reply:
Fixed, now there is only one commit containing my changes. @marseel